server/aesdsocket.c

```diff
@@ -139,6 +269,7 @@ bool create_daemon()
  */
 void clean()
```

Comment:
Not sure `exit(0)` is the right thing to do from this function in all cases where it's called.  Would be better to have a common path to fall through for error states and cleanup which still allows returning error status.